recipe/conda_build_config.yaml

```diff
@@ -1,9 +1,3 @@
-channel_targets:
-  - conda-forge llvm_rc
```

Comment:
This needs to be readded

Reply:
Ah, thanks, I haven't had an rc-PR so far and hat just pointed the changes on master against the branch. Thanks for catching this!